### PR TITLE
On plugin pull errors, delete created dirs.

### PR DESCRIPTION
**Pull request by anusha-ragunathan**
Signed-off-by: Anusha Ragunathan <anusha@docker.com>

### CODE REVIEW

File: plugin/backend_linux.go [hunk 1]
```diff
@@ -60,6 +61,29 @@ func (pm *Manager) Inspect(name string) (tp types.Plugin, err error) {
 	return p.PluginObj, nil
 }
 
+func (pm *Manager) pullInt(ref reference.Named, metaHeader http.Header, authConfig *types.AuthConfig, pluginID string) (types.PluginPrivileges, error) {
```
Comment (tiborvass) [inline plugin/backend_linux.go]: What does `pullInt` stand for?

Comment (anusha-ragunathan) [inline plugin/backend_linux.go]: Done. It was outta habit from my C world :)

**Comment by tiborvass** [comment]
LGTM

**Comment by vieux** [comment]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

**Pull request merged**